LocalPackages/SyncUI/Sources/SyncUI/Views/DeviceConnectedView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  DeviceConnectedView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import DuckUI
+
+public struct DeviceConnectedView: View {
+
+    @State var showRecoveryPDF = false
+
+    let saveRecoveryKeyViewModel: SaveRecoveryKeyViewModel
+
+    public init(saveRecoveryKeyViewModel: SaveRecoveryKeyViewModel) {
+        self.saveRecoveryKeyViewModel = saveRecoveryKeyViewModel
+    }
+
+    @ViewBuilder
+    func deviceSyncedView() -> some View {
+        VStack(spacing: 0) {
+            Image("SyncSuccess")
+                .padding(.bottom, 20)
+
+            Text(UserText.deviceSyncedTitle)
+                .font(.system(size: 28, weight: .bold))
+                .padding(.bottom, 24)
+
+            ZStack {
+                RoundedRectangle(cornerRadius: 8)
+                    .stroke(.black.opacity(0.14))
+
+                HStack(spacing: 0) {
+                    Image(systemName: "checkmark.circle")
+                        .padding(.horizontal, 18)
+                    Text("WIP: Another Device")
+                    Spacer()
+                }
+            }
+            .frame(height: 44)
+            .padding(.horizontal, 20)
+            .padding(.bottom, 20)
+
+            Text(UserText.deviceSyncedMessage)
+                .lineLimit(nil)
+                .multilineTextAlignment(.center)
+
+            Spacer()
+
+            Button {
+                withAnimation {
+                    self.showRecoveryPDF = true
+                }
+            } label: {
+                Text(UserText.nextButtonTitle)
+            }
+            .buttonStyle(PrimaryButtonStyle())
+        }
+        .padding(.top, 56)
+        .padding(.horizontal)
```

Comment:
Add also `.padding(.bottom)` to fix iPad sheet. It should also work fine for iPhone, where currently the bottom edge of the button touches safe area.

![IMG_6FFEE8E8C20B-1](https://user-images.githubusercontent.com/14022/222405253-64f4cbea-0652-4916-9933-48e7f2731eec.jpeg)